### PR TITLE
Add peppermint@1.4.0 w/ git auto-update

### LINKED ISSUE #10599: [Request] Add Peppermint
**Library name:** Peppermint
**Git repository url:** https://github.com/wilddeer/Peppermint
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:** http://wd.dizaina.net/en/scripts/peppermint/
**Wanna say something? Leave message here:**

cc- #10165 
@kennynaoh
please help me review this issue! thanks!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #10599
Related issue(s): #10856

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/wilddeer/Peppermint
 * Official website (optional, not the repository): https://wd.dizaina.net/en/scripts/peppermint/
 * NPM package url (optional):
 * License and its reference: [MIT](https://github.com/wilddeer/Peppermint/blob/master/README.md#license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 11
   - Count of stars: 156
   - Count of forks: 22

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #10599, cc @wilddeer, cc #10856

### CODE REVIEW

**Comment by qwe19272375** [comment]
@cdnjs/intern3 please help me review this PR, thanks.

**Comment by PeterDaveHello** [comment]
@qwe19272375 wonrg PR title.

**Comment by qwe19272375** [comment]
Sorry, PR title changed.

**Comment by qwe19272375** [comment]
Changed as requested.
@cdnjs/intern3 please help me review it, thanks!

File: ajax/libs/peppermint/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "library-agnostic"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wilddeer/Peppermint.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*.+(pure.js|pure.min.js))"
```
Comment (sufuf3) [inline ajax/libs/peppermint/package.json]: the result of this fileMap as the followings:
```
$ node
> glob = require('glob')
> glob.sync('**/!(*.+(pure.js|pure.min.js))')
[ 'peppermint.js',
  'peppermint.min.js',
  'peppermint.required.css',
  'peppermint.suggested.css',
  'pure' ]
```

I don't think we need the folder `pure`
How about this?
```
> glob.sync('**/!(*pure*)')
[ 'peppermint.js',
  'peppermint.min.js',
  'peppermint.required.css',
  'peppermint.suggested.css' ]
```

**Comment by qwe19272375** [comment]
@sufuf3 thanks for your advice.
@cdnjs/intern3 please help me review this PR, thank you.

**Comment by qwe19272375** [comment]
Regenerate .min files without .map files.
@cdnjs/intern3 please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@qwe19272375 don't forget to request review again as above.

**Comment by PeterDaveHello** [comment]
Please don't duplicate the commit message (the second commit), we don't need to repeat the first line.

**Pull request merged**